### ISSUE TITLE
Do you solve it?

### ISSUE THREAD

**Issue by DanteAndroid** [karray/AndroidToolbarScrolling#1]
how? Thx

**Comment by karray** [comment]
No, unfortunately I didn't. I've created a pull request somewhere in android branch, but it seems they's ignored it.

Let me know if you find any solution.
